legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/depot/auth/VoidAuthClientInjector.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.depot.auth;
+
+import org.apache.http.impl.client.HttpClientBuilder;
+
+public class VoidAuthClientInjector implements AuthClientInjector
```

Comment:
Is this class necessary anymore? Since you can basically do this now with a simple identity lambda (b -> b).

Reply:
good point, thanks I removed this class